src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/StringValidators.java

```diff
@@ -84,4 +84,13 @@ public static void checkIsEmpty(String toTest, String qualifiedFieldName, Errors
             ApiLogger.infoContext(loggingContext, qualifiedFieldName + " Field should not be populated");
         }
     }
+
+    public static boolean checkIsNotEqual(String nationality, String secondNationality, String errorMsg, String qualifiedFieldName, Errors errors, String loggingContext) {
```

Comment:
the checkIsNotEqual method is generic so should use generic variable and param names.
I'd change String nationality, String secondNationality to String string1, String string2.

Reply:
if you use StringUtils.equals() in here you won't have to add a null check, otherwise you should check strings are null as could get null pointer exception calling .equals on the string

Reply:
Names updated.